odoo/addons/base/views/res_bank_views.xml

```diff
@@ -84,19 +84,21 @@
                     <group>
                         <group>
                             <field name="acc_number"/>
-                            <field name="bank_id"/>
                             <field name="clearing_number"/>
                             <field name="partner_id"/>
+                            <field name="bank_id" options='{"no_open": True}'/>
```

Comment:
Seems like a false good idea, because we could want to open the bank in order to find other pieces of data than just the address IMO.